lib/memo_wise.rb

```diff
@@ -299,7 +296,7 @@ def memo_wise(method_name_or_hash) # rubocop:disable Metrics/PerceivedComplexity
         if method.arity.zero?
           klass.module_eval <<-END_OF_METHOD, __FILE__, __LINE__ + 1
             # def foo
-            #   @_memo_wise.fetch(:foo}) do
+            #   @_memo_wise.fetch(:foo) do
```

Comment:
Fix a pre-existing typo in the comment demonstrating what the resulting code is supposed to look like